doc/examples/transform/plot_rescale.py

```diff
@@ -3,16 +3,20 @@
 Rescale, resize, and downscale
 ==============================
 
-`Rescale` operation resizes an image by a given scaling factor.
-The scaling factor can either be a single floating point value,
-or multiple values - one along each axis.
+`Rescale` operation resizes an image by a given scaling factor. The scaling
+factor can either be a single floating point value, or multiple values - one
+along each axis.
 
-`Resize` serves the same purpose, but allows to specify an output
-image shape instead of a scaling factor.
+`Resize` serves the same purpose, but allows to specify an output image shape
+instead of a scaling factor.
+
+`Downscale` serves the purpose of downsampling an n-dimensional image by
+calculating local mean on the elements of each block of the size factors given
+as a parameter to the function.
+
+Note that when downsampling an image, `resize` and `rescale` should perform
+Gaussian smoothing to avoid aliasing artifacts.
```

Comment:
It's not clear from this whether the users should do this manually, or whether we provide a function to do so. Perhaps add the sentence, "See the `anti_aliasing` and `anti_aliasing_stddev` arguments to these functions.

And perhaps one of the calls in the example should use the new kwargs?

Reply:
Done.